### PR TITLE
[DATA-637] update model proto

### PR DESCRIPTION
**Pull request by alexis-wei**
Add getting model Info to the current model service and update to also provide file information in the metadata

### CODE REVIEW

File: proto/viam/app/model/v1/model.proto [hunk 1]
```diff
@@ -10,18 +10,26 @@ service ModelService {
...
+  rpc Info(viam.app.model.v1.InfoRequest) returns (viam.app.model.v1.InfoResponse);
 }
 
 message FileData {
   bytes data = 1;
 }
 
+message File {
+  string name = 1;
+  int64 size = 2;
```
Comment (agreenb) [inline proto/viam/app/model/v1/model.proto:22]: nit: size_bytes

File: proto/viam/app/model/v1/model.proto [hunk 2]
```diff
@@ -10,18 +10,26 @@ service ModelService {
...
+}
+
 message UploadMetadata {
   string org_id = 1;
   string model_name = 2;
   // TODO: Determine the format of the associated dataset. Update when it's decided
   // whether it should be by ID or name.
   string associated_dataset = 3;
+  repeated File files = 4;
+  int64 model_size = 5;
```
Comment (agreenb) [inline proto/viam/app/model/v1/model.proto:32]: nit: model_size_bytes

File: proto/viam/app/model/v1/model.proto [hunk 3]
```diff
@@ -48,6 +56,21 @@ message DeployRequest {
   DeployMetadata metadata = 1;
 }
 
+message Model {
+  string model_name = 1;
```
Comment (agreenb) [inline proto/viam/app/model/v1/model.proto:60]: Think can rename `model_name` and `model_size` to `name` and `size_bytes`

supernit: perhaps switch fields 2 and 3 to keep the above two fields as consecutive.

**Review by agreenb — CHANGES_REQUESTED** [review]
Looks great! Just a few nit requests.

File: proto/viam/app/model/v1/model.proto (hunk 1 above)
Comment (alexis-wei) [reply proto/viam/app/model/v1/model.proto]: changed!

File: proto/viam/app/model/v1/model.proto (hunk 3 above)
Comment (alexis-wei) [reply proto/viam/app/model/v1/model.proto]: changed!

**Review by agreenb — APPROVED** [review]
LGTM after making the other generated files from make build lint!

**Pull request merged**